lib/Jsrt/ChakraCore.h

```diff
@@ -163,7 +163,7 @@ JsGetModuleHostInfo(
     _In_ JsModuleHostInfoKind moduleHostInfo,
     _Outptr_result_maybenull_ void** hostInfo);
 
-#ifndef NTBUILD
+#ifdef CHAKRACOREBUILD_
```

Comment:
Unless you find out why you needed these, removing them seems cleaner to me. (The original design did not require `#ifdef` -- "ChakraCommon.h" was supposed to hold APIs common to Core and Full. "ChakraCore.h" was supposed to hold new ChakraCore APIs which might later be promoted to "ChakraCommon.h".)